system/Common.php

```diff
@@ -563,6 +563,24 @@ function function_usable(string $functionName): bool
     }
 }
 
+if (! function_exists('solidus')) {
+    /**
+     * Generates the solidus character (`/`) depending on the HTML5 compatibility flag in `Config\DocTypes`
+     *
+     * @internal
+     */
+    function solidus(?bool $html5 = null): string
+    {
+        $doctype = config('DocTypes')->html5 ?? false;
+
+        if ($html5 !== null) {
+            $doctype = $html5;
+        }
+
+        return $doctype ? '' : '/';
```

Comment:
With space is much better. But it should be done on the rendering part, not here.

Reply:
Both single space or not, is `valid` with html validator. My opinion 1 character affect the amount loaded by the browser, but that's okay because spaces only contain a small size. I follow team CI

Reply:
We need a space, because the current code outputs a space.